lib/node/runner/api/api.go

```diff
@@ -106,27 +103,3 @@ func TriggerEvent(st *storage.LevelDBBackend, transactions []*transaction.Transa
 	}
 
 }
-
-func renderEventStream(args ...interface{}) ([]byte, error) {
```

Comment:
the renderEventStream function is used only by the subscribe API
and it is impossible to get data from storage with this design.
Hence, this function is moved into subscribe API handler and let the function use the closure.